src/Api/Vault/AuthorizationHandlers/Collections/CollectionAuthorizationHandler.cs

```diff
@@ -83,95 +79,100 @@ public class CollectionAuthorizationHandler : BulkAuthorizationHandler<Collectio
     }
 
     private async Task CanCreateAsync(AuthorizationHandlerContext context, CollectionOperationRequirement requirement,
-        CurrentContextOrganization org)
+        CurrentContextOrganization? org)
```

Comment:
Same question here, do we need the nullable modifier?

Reply:
In this case its not strictly required for this method, as we don't have any explicit `org == null` in the `CanCreateAsync()` method. 

However, with the `#nullable enable` directive I found it beneficial to explicitly highlight that the `org` parameter has the potential to be null (in case of Provider users). This is also to be consistent with the other private `Can...Async()` methods that do have some `org is not null` checks.